module2/src/com/epam/sorting/SortingImplementer.java

```diff
@@ -0,0 +1,76 @@
+package com.epam.sorting;
+
+public class SortingImplementer {
```

Comment:
Let's better rename it like
```suggestion
public class ArraySorter {
```